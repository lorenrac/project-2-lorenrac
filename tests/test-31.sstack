// Test some edge case:
PRINT "Trim Back:  " + "hello" / "x"  // expect: "hello"
PRINT "Trim Front: " + "hello" % "x"  // expect: ""
PRINT "Remove:     " + "hello" - "x"  // expect: "hello"
