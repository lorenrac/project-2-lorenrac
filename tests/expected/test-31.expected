Trim Back:  hello
Trim Front: hello
Remove:     hello
